### PR TITLE
*: bump to hyperkube v1.0.7

### PR DESCRIPTION
**Pull request by bcwaldon**

### CODE REVIEW

**Comment by bcwaldon** [comment]
Corresponding coreos-overlay PR for the kubelet in CoreOS here: https://github.com/coreos/coreos-overlay/pull/1618

File: Documentation/deploy-master.md [hunk 1]
```diff
@@ -405,7 +405,7 @@ A successful response should look something like:
 {
   "major": "1",
   "minor": "0",
-  "gitVersion": "v1.0.6",
+  "gitVersion": "v1.0.7",
   "gitCommit": "388061f00f0d9e4d641f9ed4971c775e1654579d",
```
Comment (aaronlevy) [inline Documentation/deploy-master.md]: nit: this hash will also be changed...

Comment (bcwaldon) [inline Documentation/deploy-master.md]: i honestly don't care about this output being kept up to date. How about we just stop modifying it as of v1.0.6?

Comment (aaronlevy) [inline Documentation/deploy-master.md]: Fine with me. Also could just put in an obvious placeholder.

**Comment by aaronlevy** [comment]
LGTM

**Pull request merged**